js/btcalpha.js

```diff
@@ -645,7 +645,8 @@ module.exports = class btcalpha extends Exchange {
             throw new InvalidOrder (this.id + ' ' + this.json (response));
         }
         const order = this.parseOrder (response, market);
-        amount = (order['amount'] > 0) ? order['amount'] : amount;
+        const orderAmount = order['amount'].toString ();
```

Comment:
@samgermain is this a leftover or something?

If not, we can't do `order['amount'].toString ();` or even extend a parsed order with a stringified value

Reply:
> @samgermain is this a leftover or something?

What do you mean when you say that?

> If not, we can't do `order['amount'].toString ();` or even extend a parsed order with a stringified value

why not?

Reply:
@carlosmiei 👋